### ISSUE TITLE
cache all songs on initial load

### ISSUE THREAD

**Issue by ConcernedHobbit** [TKOaly/laulum.me#79]
download all songs' data to cache to allow offline viewing from first page load.

**Comment by JaniL** [comment]
How valuable this is actually? I'm asking because I was considering taking a look at this.

After having a short discussion about this with @hekazu I have a concern if we can justify downloading 868kb of songs on the first load.

**Comment by JaniL** [comment]
So this is actually happening already - all songs are loaded with the default next-pwa configuration.

However I was unable to open the front page while being offline.

**Comment by ConcernedHobbit** [comment]
yes, the pwa seems to have trouble actually accessing the content. would need to do an in-depth dive on how the pwa (and associated service worker) works to figure out how to get it to function properly
